test/integration/dockerized_job_conf.yml

```diff
@@ -0,0 +1,28 @@
+---
+runners:
+  local:
+    load: galaxy.jobs.runners.local:LocalJobRunner
+    workers: 1
+
+execution:
+  default: local_docker
+  environments:
+    local_docker:
+      runner: local
+      docker_enabled: true
+      docker_sudo: false
+    local_docker_inline_container_resolvers:
+      runner: local
+      docker_enabled: true
+      docker_sudo: false
```

Comment:
I think I tried to make this the default with the YAML. Might want to drop this and see if it works.